pkg/apis/core/types_shoot.go

```diff
@@ -431,6 +431,8 @@ type Kubernetes struct {
 	// Kubelet contains configuration settings for the kubelet.
 	Kubelet *KubeletConfig
 	// Version is the semantic Kubernetes version to use for the Shoot cluster.
+	// Defaults to the highest supported minor and patch version given in the referenced cloud profile.
+	// The version can be omitted completely or partially specified, e.g. `<major>.<minor>`.
 	Version string
```

Comment:
pointer?

Reply:
Please see eb3963710042358feabb98121d3c2833928bb9d2 😓 
I also tested this change WRT backwards compatibility and Protobuf: An "older" Gardenlet version was still able to connect to a "new" Apiserver ✅

Reply:
I dropped the commit again after our discussion and added a note why we stick to `string` 🙂